mobile/components/FormCard.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { useState } from "react";
+import { View, styled } from "tamagui";
+import Badge, { BadgeProps } from "./Badge";
+import Card from "./Card";
+import { Typography } from "./Typography";
+import CardFooter from "./CardFooter";
+
+export interface FormCardProps {
+  /**
+   * Header text
+   */
+  header: string;
+
+  /**
+   * Subheader optional text
+   */
+  subHeader?: string;
+
+  /**
+   * Footer text
+   */
+  footer: string;
+
+  /**
+   * Optional preset type.
+   */
+  badgeProps: BadgeProps;
```

Comment:
i think you forgot to add the `?` for the optional prop

Reply:
In case badgeProps is not passed, should we display the default "Not started" badge or not to display any badge at all?

Reply:
I would say display "Not started' as default